Assets/GothicVR/Scripts/Player/Camera/CameraFade.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace GVR
+{
+    public class CameraFade : MonoBehaviour
+    {
+        public Image cameraFadeImage;
+
+        private void Start()
+        {
+            Fade(0.1f, 0);
+        }
+
+        public void Fade(float duration, float targetAlpha)
+        {
+            StartCoroutine(FadeCamera(duration, targetAlpha));
+        }
+
+        private IEnumerator FadeCamera(float duration, float targetAlpha)
+        {
+            float currentTime = 0;
+
+            while (currentTime < duration)
+            {
+                currentTime += Time.fixedDeltaTime;
```

Comment:
I think you created an issue here.
Coroutines will be called every frame. Then you should use Time.deltaTime to set value every frame.

But! If you want to use Fixed (physics based?) updates, then you can optimize like this (I read on the forum):
```c#
currentTime += Time.fixedDeltaTime;
... // do your magic
yield return WaitForFixedUpdate(); // Now coroutine gets called again when we really want it. Ensures we skip unneccesary frames.
```

Reply:
pretty sure it was supposed to be deltaTime, dont know how fixed got in there xD